doc/Development-Guide-Rad-Tool.md

```diff
@@ -3,43 +3,46 @@
   
 ### Introduction
 
- In this document, we will introduce **ASP.NET Zero Power Tools** and explain it. This tool is developed to minimize the effort of creating a new entity. It creates all related layers (including UI) by defining an entity.
+ In this document, we will introduce **ASP.NET Zero Power Tools** and explain it. This tool is developed to minimize the effort of creating a new entity. It creates all the related layers (including UI) by defining an entity.
 
 ### Download And Install
 
- If your project version is 5.1.0+, all you have to do is just installing **ASP.NET Zero Power Tools** extension on visual studio, from https://marketplace.visualstudio.com/items?itemName=Volosoft.AspNetZeroPowerTools.
+ If your project is version 5.1.0+, all you have to do is just instal the **ASP.NET Zero Power Tools** extension on Visual Studio from https://marketplace.visualstudio.com/items?itemName=Volosoft.AspNetZeroPowerTools.
```

Comment:
- "If your project `version` is ~`version`~ `v`5.1.0+" reads better.
- ~`instal`~`install`

Reply:
This doesn't feel right to me, but I haven't been able to find any sources that dictate a style and it's minor so I'll make the changes.